### PR TITLE
stats: fix row count estimation for null

### PR DESCRIPTION
**Pull request by lamxTyler**
When building columns stats, null values will only be recorded in `NullCount`, so it will be wrong to use the histogram or CM sketch to estimate. For index stats, we should not skip the null values when building the scan range.
PTAL @coocood @winoros @zz-jason

### CODE REVIEW

**Comment by zz-jason** [comment]
@lamxTyler sqllogic-test failed

**Comment by shenli** [comment]
Please fix the CI.

**Comment by winoros** [comment]
It seems that all plans' is MaxFloat64 so `convert2PhysicalPlan` returns `nil.

**Comment by lamxTyler** [comment]
PTAL @zz-jason @winoros

**Comment by lamxTyler** [comment]
PTAL @zz-jason @winoros

File: statistics/table.go [hunk 1]
```diff
@@ -252,7 +252,7 @@ func (t *Table) ColumnIsInvalid(sc *stmtctx.StatementContext, colID int64) bool
 		sc.SetHistogramsNotLoad()
 		histogramNeededColumns.insert(tableColumnID{tableID: t.TableID, columnID: colID})
 	}
-	return !ok || (col.NDV > 0 && col.Len() == 0)
+	return !ok || col.totalRowCount() == 0 || (col.NDV > 0 && col.Len() == 0)
```
Comment (winoros) [inline statistics/table.go]: This is the case that the column only have null values?

**Pull request merged**